components/kubernetes/kind.go

```diff
@@ -35,10 +34,17 @@ func NewKindCluster(vm *vm.UnixVM, clusterName, arch string) (*remote.Command, p
 	if err != nil {
 		return nil, pulumi.StringOutput{}, err
 	}
+
+	kindVersionConfig, found := kubeToKindVersion[KubernetesVersion]
+
+	if !found {
```

Comment:
I don't think you need to limit this. If version is not found, use latest Kind version and generate the image with the requested string version

Reply:
What if the user asks for kubernetes version `1.18` ? 

If I default to the latest Kind version, then I won't be able to spin a cluster with kubernetes version `1.18` because the latest version of kind doesn't support `1.19` version of kubernetes.

So I will either leave it fail to create the cluster, and the user will have to understand what's going on. Or, I can simply tell the user that this is not going to work from the beginning and show them what kube versions they can use. I thought the latter would be more convenient.

WDYT?